tests/integration_tests/queries/saved_queries/api_tests.py

```diff
@@ -157,12 +157,12 @@ def test_get_list_saved_query_gamma(self):
         """
         Saved Query API: Test get list saved query
         """
-        gamma = self.get_user("gamma")
+        user = self.get_user("gamma_sqllab")
```

Comment:
The `gamma_sqllab` test user has both `Gamma` and `sql_lab` roles.

Reply:
did we had to change this user for the tests to pass?